### PR TITLE
[MDS-52] Merge tariff publishing document urls with tariff rates

### PR DESCRIPTION
**Pull request by lipsotiko**

### CODE REVIEW

File: TariffRatesBlobTrigger/unit_tests.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
+import unittest
+import translator
+import csv
+import json
+from io import StringIO
+
+
+class TestStringMethods(unittest.TestCase):
+    def test_tariff_with_publishing_document(self):
+        publishing_docs_json = open('mocks/sample_traiff_pubications.json')
```
Comment (loren) [inline TariffRatesBlobTrigger/unit_tests.py:10]: s/traiff/tariff

File: TariffRatesBlobTrigger/azure_auth.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+import requests
+
+
+auth_url = os.environ["AZURE_AUTH_URL"]
+client_id = os.environ["TARIFF_DOCS_CLIENT_ID"]
+client_secret = os.environ["TARIFF_DOCS_CLIENT_SECRET"]
+
+
+def get_access_token():
+    payload = '''grant_type=client_credentials&
```
Comment (loren) [inline TariffRatesBlobTrigger/azure_auth.py:12]: maybe an f string here instead?

File: README.md [hunk 3]
```diff
@@ -22,9 +22,13 @@ It's git ignored because it will contain access keys
 {
```
Comment (loren) [inline README.md:22]: Not part of your PR but `local.settins.json` is misspelled right above this, and above that the prerequisites section needs to be modified for Azure

File: .flake8 [hunk 4]
```diff
@@ -0,0 +1,2 @@
+[flake8]
+max-line-length = 160
```
Comment (loren) [inline .flake8:2]: seems awfully long to me...just making sure this wasn't a typo!

File: requirements.txt [hunk 5]
```diff
@@ -1 +1,4 @@
-azure-functions
\ No newline at end of file
+azure-functions
+azure-storage
+azure-storage-blob
+flake8
```
Comment (loren) [inline requirements.txt:4]: Maybe break out test requirements from dev requirements?

File: TariffRatesBlobTrigger/unit_tests.py (hunk 1 above)
Comment (lipsotiko) [reply TariffRatesBlobTrigger/unit_tests.py:10]: What do you mean here?

File: .flake8 (hunk 4 above)
Comment (lipsotiko) [reply .flake8:2]: I have a large monitor, I switched it to 120

File: TariffRatesBlobTrigger/unit_tests.py (hunk 1 above)
Comment (loren) [reply TariffRatesBlobTrigger/unit_tests.py:10]: I think the filename in that line is misspelled.

**Pull request merged**